.github/shared/test/exec.test.js

```diff
@@ -20,13 +22,13 @@ describe("execFile", () => {
 
   it("exec succeeds with exact-sized buffer", async () => {
     await expect(
-      execFile(file, args, { maxBuffer: expected.length }),
+      execFile(file, args, { maxBuffer: expected.length, ...options }),
```

Comment:
```suggestion
      execFile(file, args, {  ...options, maxBuffer: expected.length }),
```

NIT: last value wins, so better to spread defaults first